compiler/src/dotty/tools/dotc/decompiler/IDEDecompilerDriver.scala

```diff
@@ -19,7 +19,9 @@ class IDEDecompilerDriver(val settings: List[String]) extends dotc.Driver {
     val rootCtx = initCtx.fresh.addMode(Mode.Interactive | Mode.ReadPositions | Mode.ReadComments)
     rootCtx.setSetting(rootCtx.settings.YretainTrees, true)
     rootCtx.setSetting(rootCtx.settings.fromTasty, true)
-    val ctx = setup(settings.toArray :+ "dummy.scala", rootCtx)._2
+    val ctx = setup(settings.toArray :+ "dummy.scala", rootCtx) match
```

Comment:
We should never be passing `--help` here or another flag that interrupts compilation so you could just do `.get` on the result of `setup`.

Reply:
Will change